src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-list/iscsi-target-list.component.ts

```diff
@@ -124,29 +128,49 @@ export class IscsiTargetListComponent extends ListWithDetails implements OnInit,
     this.iscsiService.status().subscribe((result: any) => {
       this.available = result.available;
 
-      if (result.available) {
-        this.iscsiService.version().subscribe((res: any) => {
-          this.cephIscsiConfigVersion = res['ceph_iscsi_config_version'];
-          this.taskListService.init(
-            () => this.iscsiService.listTargets(),
-            (resp) => this.prepareResponse(resp),
-            (targets) => (this.targets = targets),
-            () => this.onFetchError(),
-            this.taskFilter,
-            this.itemFilter,
-            this.builders
-          );
-        });
-
-        this.iscsiService.settings().subscribe((settings: any) => {
-          this.settings = settings;
-        });
-      } else {
+      if (!result.available) {
         this.status = result.message;
       }
     });
   }
 
+  timeConditionReached() {
```

Comment:
Following on our clean code DRY agreement and taking int oaccount tahta we have already have 2 identical `timeConditionReached` declarations, please extract this method for reusability (i.e. declaring it in: src/pybind/mgr/dashboard/frontend/src/app/shared/classes/list-with-details.class.ts)

Reply:
Done.